xbmc/addons/Scraper.cpp

```diff
@@ -816,6 +816,23 @@ void DetailsFromFileItem<CArtist>(const CFileItem &item, CArtist &artist)
     artist.discography.emplace_back(discoAlbum);
   }
 
+  const int numvideolinks = item.GetProperty("artist.videolinks").asInteger32();
+  if (numvideolinks > 0)
+  {
+    artist.videolinks.reserve(numvideolinks);
+    for (int i = 1; i < numvideolinks; ++i)
```

Comment:
`i <= numvideolinks` - otherwise you will loose last link!

Reply:
Thanks!